app/lib/bk/compat/parsers/bitrise/steps.rb

```diff
@@ -28,6 +59,18 @@ def translate_git_clone(_inputs)
           '# No need for cloning, the agent takes care of that'
         end
 
+        def translate_git_tag(inputs)
+          return '# Invalid git-tag step configuration!' unless inputs.include?('tag')
+
+          cmd_all_params = "git tag -fa #{inputs['tag']} -m \"#{inputs['tag_message']}\""
+          cmd_no_message = "git tag -fa #{inputs['tag']}"
+
+          [
+            inputs.fetch('tag_message', false) ? cmd_all_params : cmd_no_message,
+            inputs.fetch('push', false) && inputs['push'] ? 'git push --tags' : nil
```

Comment:
the `inputs['push']` is now unnecessary:

```suggestion
            inputs.fetch('push', false) ? 'git push --tags' : nil
```

Reply:
and, as mentioned in the comments, if the default is the same as when the value is not present, the `fetch` is unnecessary:

```suggestion
            inputs['push'] ? 'git push --tags' : nil
```

Reply:
One I missed - cheers 👍